config/filter.d/zzz-generic-example.conf

```diff
@@ -0,0 +1,17 @@
+# Fail2Ban generic example resp. test filter 
```

Comment:
Because we should build then a biggish test case (that doing the same as our testSampleRegexs-Builder).
Or we say now easy: the filters begins with zzz-* are example's or just for test cases.

I'll check later how I can extract the testSampleRegexs-Builder functionality to separate test case with some common method... Then we can move it to the tests-directory. Currently impossible, because of include also.

And now we have 2 major branches... Hmm